src/Symfony/Component/Console/Command/CommandInterface.php

```diff
@@ -0,0 +1,305 @@
+     * like a daemon.
+     *
+     * PHP 5.5+ or the proctitle PECL library is required
+     *
+     * @param string $title The process title
+     *
+     * @return $this
+     */
+    public function setProcessTitle($title);
+
+    /**
+     * Returns the command name.
+     *
+     * @return string The command name
+     */
+    public function getName();
+
+    /**
+     * Sets the description for the command.
+     *
+     * @param string $description The description for the command
+     *
+     * @return $this
+     */
+    public function setDescription($description);
+
+    /**
+     * Returns the description for the command.
+     *
+     * @return string The description for the command
+     */
+    public function getDescription();
+
+    /**
+     * Sets the help for the command.
+     *
+     * @param string $help The help for the command
+     *
+     * @return $this
+     */
+    public function setHelp($help);
+
+    /**
+     * Returns the help for the command.
+     *
+     * @return string The help for the command
+     */
+    public function getHelp();
+
+    /**
+     * Returns the processed help for the command replacing the %command.name% and
+     * %command.full_name% patterns with the real values dynamically.
+     *
+     * @return string The processed help for the command
+     */
+    public function getProcessedHelp();
+
+    /**
+     * Sets the aliases for the command.
+     *
+     * @param string[] $aliases An array of aliases for the command
+     *
+     * @return $this
+     *
+     * @throws \InvalidArgumentException When an alias is invalid
+     */
+    public function setAliases($aliases);
+
+    /**
+     * Returns the aliases for the command.
+     *
+     * @return array An array of aliases for the command
+     */
+    public function getAliases();
+
+    /**
+     * Returns the synopsis for the command.
+     *
+     * @param bool $short Whether to show the short version of the synopsis (with options folded) or not
+     *
+     * @return string The synopsis
+     */
+    public function getSynopsis($short = false);
+
+    /**
+     * Add a command usage example.
+     *
+     * @param string $usage The usage, it'll be prefixed with the command name
+     *
+     * @return $this
+     */
+    public function addUsage($usage);
+
+    /**
+     * Returns alternative usages of the command.
+     *
+     * @return array
+     */
+    public function getUsages();
+
+    /**
+     * Gets a helper instance by name.
+     *
+     * @param string $name The helper name
+     *
+     * @return mixed The helper value
+     *
+     * @throws \LogicException           if no HelperSet is defined
+     * @throws \InvalidArgumentException if the helper is not defined
+     */
+    public function getHelper($name);
+
+    /**
+     * Returns a text representation of the command.
+     *
+     * @return string A string representing the command
+     *
+     * @deprecated since version 2.3, to be removed in 3.0.
```

Comment:
It see you've based your PR on 2.7. 2.7 is not supported anymore and new features go into the master. Can you change your PR to be against the master instead? Github will let you change it, but you will still need to change the base branch in your git branch.